htdocs/compta/facture/card.php

```diff
@@ -4476,7 +4476,7 @@ function js_recalculate_revenuestamp(){
 	// Show global modifiers
 	if (! empty($conf->global->INVOICE_USE_SITUATION))
 	{
-		if ($object->situation_cycle_ref && $object->statut == 0) {
+		if ($object->statut == 0) { //SITUATION can add lines during cycle a additional works
```

Comment:
Why removing test on situation_cycle_ref ?
Do you mean the form after is show even if the invoice is not a situation invoice ?